src/main/java/com/google/cloud/pontem/CloudSpannerDatabaseBackup.java

```diff
@@ -229,6 +229,13 @@
     String[] getTablesToExcludeFromBackup();
 
     void setTablesToExcludeFromBackup(String[] value);
+
+    @Description("Cloud Spanner host")
+    @Required
+    @Default.String(Util.CLOUD_SPANNER_API_ENDPOINT_HOSTNAME)
+    String getSpannerHost();
+
+   void setSpannerHost(String value);
```

Comment:
Nit: seems that the indentation is missing a whitespace. I'm using a mobile device so this may be me.